content/updates/2020-06/1-duplicate-a-product.md

```diff
@@ -0,0 +1,13 @@
+# New: duplicate a product
+::: meta-data type="New" features="Enrichment, Product creation" available="early-June" link-to-doc="../articles/duplicate-a-product.html"
+:::
+
+Do you want to easily create a new product from another? You can now duplicate a simple product in your user interface with one single click! What a time-saver! :wink:  
+Find this new action under the `...` button at the top right corner of each of your product edit forms. 
```

Comment:
```suggestion
Find this new action under the `...` button in the top right corner of each of your product edit forms. 
```

Reply:
I'm not sure about the 'each of your pef' -> would it be accurate to say in the top right corner of the pef (singular)? if It works, it'd be lighter!

Reply:
Yes, that'd work!